clang/include/clang/Serialization/ASTReader.h

```diff
@@ -1442,6 +1442,10 @@ class ASTReader
     const StringRef &operator*() && = delete;
   };
 
+  /// VarDecls with initializers containing side effects must be emitted,
+  /// but DeclMustBeEmitted is not allowed to deserialize the intializer.
+  llvm::SmallPtrSet<Decl *, 16> InitSideEffectVars;
```

Comment:
Not a big deal, but do we have any kind of data informing us how many of these we typically see?